src/axom/inlet/tests/inlet_function.cpp

```diff
@@ -54,11 +55,10 @@ TEST(inlet_function, simple_vec3_to_vec3_raw)
   auto inlet = createBasicInlet(&ds, testString);
 
   auto func =
-    inlet.reader().getFunction("foo", FunctionType::Vec3D, {FunctionType::Vec3D});
+    inlet.reader().getFunction("foo", FunctionTag::Vec3D, {FunctionTag::Vec3D});
 
   EXPECT_TRUE(func);
-  auto result =
-    func.call<axom::primal::Vector3D>(axom::primal::Vector3D {1, 2, 3});
+  auto result = func.call<FunctionType::Vec3D>(FunctionType::Vec3D {1, 2, 3});
```

Comment:
Would this function also work if you only supplied 1 or 2 parameters?
e.g.
```cpp
   auto result = func.call<FunctionType::Vec3D>(FunctionType::Vec3D {1});      --> [2,0,0]
   auto result = func.call<FunctionType::Vec3D>(FunctionType::Vec3D {1, 2});   --> [2,4,0]
```

...or, are three parameters always required?

Reply:
The `Vector` constructor will value-initialize the remaining components when less than three are provided - I'll add a test for this.

Reply:
Thanks @joshessman-llnl -- that's what I thought